frontend/lib/src/controllers/spending_breakdown_controller.dart

```diff
@@ -16,14 +18,22 @@ class SpendingBreakdownController extends GetxController {
 
   set setTotalSpent(double newValue) => _totalSpent.value = newValue;
 
-  // TODO: replace during integration of Spending Breakdown
-  getSpendingBreakdown() {
+  void getSpendingBreakdown({String accountId = '', int days = 0}) async {
+    accountId = accountId;
     var totalExpenses = 0.0;
+    spendingList.value = await AccountService.getBreakdown(
+      accountId: accountId,
+      days: days,
+    );
 
-    for (var breakdown in spendingBreakdownData) {
+    for (var breakdown in spendingList) {
       totalExpenses += breakdown.totalTransactionAmount;
     }
 
     _totalSpent.value = totalExpenses;
   }
+
+  void resetState() {
```

Comment:
```suggestion
  void resetFilters() {
```
this can be misleading